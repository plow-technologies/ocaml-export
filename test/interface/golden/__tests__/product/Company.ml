let () =
  AesonSpec.sampleGoldenAndServerSpec Company.decodePerson Company.encodePerson "person" "http://localhost:8081/person" "__tests__/golden/Person.json";

  AesonSpec.sampleGoldenAndServerSpec Company.decodeCompany Company.encodeCompany "company" "http://localhost:8081/company" "__tests__/golden/Company.json";
