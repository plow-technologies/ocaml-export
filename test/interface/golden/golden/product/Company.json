{
    "seed": 9137208507124092156,
    "samples": [
        {
            "address": "\u000f8h'rtï\u0002w{w·\"Jn\u0003.H\u0018\u001531",
            "employees": [
                {
                    "created": "1864-04-26T00:00:01.011Z",
                    "name": ",\u0005Neé\u0017|",
                    "id": 30
                }
            ]
        },
        {
            "address": "b\tôq\u0014+(ß",
            "employees": [
                {
                    "created": "1864-04-23T00:00:01.011Z",
                    "name": "NÂqôÑãKAíî\u0017\u0002\u001bªD\u0011\ta\u0004nÇ¾\u000c\u001a",
                    "id": 11
                },
                {
                    "created": "1864-05-31T00:00:01.011Z",
                    "name": "l",
                    "id": -5
                },
                {
                    "created": "1864-05-29T00:00:01.011Z",
                    "name": "k}>\u001eL!z*\u0005\u0004\"i¾ÚùU\u0006n1\u0008\u0019hg£",
                    "id": 27
                },
                {
                    "created": "1864-04-14T00:00:01.011Z",
                    "name": null,
                    "id": 18
                },
                {
                    "created": "1864-04-27T00:00:01.011Z",
                    "name": null,
                    "id": 26
                },
                {
                    "created": "1864-05-23T00:00:01.011Z",
                    "name": null,
                    "id": 25
                },
                {
                    "created": "1864-05-02T00:00:01.011Z",
                    "name": "Ëw&\u0006BMKVVZR\u001f>ò\u0003g\u0008F<A3FSÔ",
                    "id": 23
                },
                {
                    "created": "1864-05-10T00:00:01.011Z",
                    "name": "1d",
                    "id": 24
                },
                {
                    "created": "1864-06-02T00:00:01.011Z",
                    "name": "Epøq~IM\u0003>²I@è",
                    "id": -5
                },
                {
                    "created": "1864-05-01T00:00:01.011Z",
                    "name": null,
                    "id": 18
                },
                {
                    "created": "1864-06-08T00:00:01.011Z",
                    "name": "\u0015\u0015¿tÂW{",
                    "id": 27
                },
                {
                    "created": "1864-04-26T00:00:01.011Z",
                    "name": " c\u0003><Ô\u001fè\u0019\u001f\u0017Ñß]F\u0018U\u0002\u0011|Q.\u0003¶­ ",
                    "id": 22
                }
            ]
        },
        {
            "address": "\u001f\u0008ü2;.R\u001e!\u001cs½5vxv\u0016tIcX\nì",
            "employees": [
                {
                    "created": "1864-04-09T00:00:01.011Z",
                    "name": null,
                    "id": -22
                },
                {
                    "created": "1864-04-25T00:00:01.011Z",
                    "name": " ",
                    "id": 16
                },
                {
                    "created": "1864-06-05T00:00:01.011Z",
                    "name": "o<#W\rë³$\u000b{¹k*}Y14-q-oyæ/",
                    "id": -9
                },
                {
                    "created": "1864-04-23T00:00:01.011Z",
                    "name": "uM$t×WÃqf\u0005K3WÿÀ\u000721({u\u0016$\u0002",
                    "id": 19
                },
                {
                    "created": "1864-05-09T00:00:01.011Z",
                    "name": "gF\u0016'MØ,\n",
                    "id": -21
                },
                {
                    "created": "1864-04-25T00:00:01.011Z",
                    "name": "Ï\u0000az\u0011\u0019\u0008\t³Yß\u0018q-wÝ h3ïÝ=ó{X",
                    "id": 14
                },
                {
                    "created": "1864-05-24T00:00:01.011Z",
                    "name": "KX\u000ex\u0015PW\u0018",
                    "id": -27
                },
                {
                    "created": "1864-06-06T00:00:01.011Z",
                    "name": "\u001a*>$k_\u001aÿ",
                    "id": 18
                },
                {
                    "created": "1864-05-31T00:00:01.011Z",
                    "name": "}s,²ÏÉ$½6`m\u000bÁ7r8f\u0011$Uxq~H*Da",
                    "id": -21
                },
                {
                    "created": "1864-05-04T00:00:01.011Z",
                    "name": "K09@~iK\u0005(\u0010ÿ{\u000eQm\u0015Vö\u0010QDC\u001bkA\"\u000b}^",
                    "id": 18
                },
                {
                    "created": "1864-04-18T00:00:01.011Z",
                    "name": ",",
                    "id": -5
                },
                {
                    "created": "1864-05-11T00:00:01.011Z",
                    "name": "ìc¯M!qg.\u0015",
                    "id": -1
                },
                {
                    "created": "1864-05-20T00:00:01.011Z",
                    "name": "Å_ÚJ\u000em\u0014Jo',.vl?M",
                    "id": -5
                }
            ]
        },
        {
            "address": "?]#+:p\nL\u001f5%\u0016¸V_w\u0014]ä·8",
            "employees": [
                {
                    "created": "1864-06-07T00:00:01.011Z",
                    "name": "PR\u0017VH '<ùÀæMÇ0«çO]\u0007â¦Úo°®",
                    "id": 9
                }
            ]
        },
        {
            "address": "T5$hPÏýr{|5\r\u000frK'^\u001fu¦N'\u0000Ã",
            "employees": [
                {
                    "created": "1864-05-11T00:00:01.011Z",
                    "name": "Ú7KNg\u001ddGÙ6\u0011\u0006Y\u000c8|\n",
                    "id": 4
                },
                {
                    "created": "1864-05-13T00:00:01.011Z",
                    "name": "¤srB=¯\u0011\u0001ªcfM4OXò",
                    "id": 4
                },
                {
                    "created": "1864-05-09T00:00:01.011Z",
                    "name": "\u0007\u0003\u0011_\\'L\u0012Oûim*\\ÄW1¯%VzLW",
                    "id": -25
                },
                {
                    "created": "1864-04-15T00:00:01.011Z",
                    "name": "'\u0007\u0008l\\\u0000L>\u0000+\u0016Ñ2kHër\u001b\u001dnÝ\u0007¡t'",
                    "id": 10
                },
                {
                    "created": "1864-04-09T00:00:01.011Z",
                    "name": "\u001d v·(h&O3V",
                    "id": 30
                },
                {
                    "created": "1864-06-05T00:00:01.011Z",
                    "name": null,
                    "id": 15
                },
                {
                    "created": "1864-05-21T00:00:01.011Z",
                    "name": "Z\u00007Û)rÛ0\"30\u0012+",
                    "id": -9
                },
                {
                    "created": "1864-04-23T00:00:01.011Z",
                    "name": "z&1Ä'\u0014\u0019\u000f©?\u001fà",
                    "id": 26
                },
                {
                    "created": "1864-04-24T00:00:01.011Z",
                    "name": null,
                    "id": -3
                },
                {
                    "created": "1864-05-27T00:00:01.011Z",
                    "name": null,
                    "id": -18
                },
                {
                    "created": "1864-05-13T00:00:01.011Z",
                    "name": "\u0003çg\u00032ø/\u0007J( Ø\u000f\u0000@SY\u0002n",
                    "id": 13
                },
                {
                    "created": "1864-05-19T00:00:01.011Z",
                    "name": "âù`^öcnØ@+5)Í´",
                    "id": 26
                },
                {
                    "created": "1864-05-20T00:00:01.011Z",
                    "name": "+\u0006\t>\t_\\ô`\u0019E³\u001cÅQ5",
                    "id": 14
                },
                {
                    "created": "1864-04-11T00:00:01.011Z",
                    "name": "'LÀIb#\u0010\u0014l'hz\\oYf\u0004",
                    "id": 10
                },
                {
                    "created": "1864-04-25T00:00:01.011Z",
                    "name": "\u001f\u001fãd\u0014fÿ\u000bL`,\r.vè\u001d",
                    "id": 17
                },
                {
                    "created": "1864-05-21T00:00:01.011Z",
                    "name": "sr\u0014<Ij×­\\\u0001\\oû\u0019\u0001Kk\u001b\u0004^E\u0007b",
                    "id": -19
                },
                {
                    "created": "1864-05-06T00:00:01.011Z",
                    "name": "úÁ\u0000\u0007¸'\u001eú*4",
                    "id": -2
                },
                {
                    "created": "1864-05-09T00:00:01.011Z",
                    "name": "-î\u0014T(\u000c\u0001»ËK\u0013%u@Jî# +`Ë\u0011\u0018\u0015\u0005",
                    "id": 29
                },
                {
                    "created": "1864-05-31T00:00:01.011Z",
                    "name": ")©ÀDDËK\u001fr_í\u0018r",
                    "id": -23
                },
                {
                    "created": "1864-04-12T00:00:01.011Z",
                    "name": "hW^{Y¸jft%;F`\u001bT4WN0",
                    "id": 8
                },
                {
                    "created": "1864-04-30T00:00:01.011Z",
                    "name": null,
                    "id": -24
                },
                {
                    "created": "1864-04-17T00:00:01.011Z",
                    "name": "$gu th=\u001eÂ4Õ<Bî\u0006\u001c",
                    "id": 25
                },
                {
                    "created": "1864-04-19T00:00:01.011Z",
                    "name": null,
                    "id": -14
                },
                {
                    "created": "1864-04-23T00:00:01.011Z",
                    "name": "~HV!\u001f*\u0019T7\"W}¡]0!àlz",
                    "id": -5
                }
            ]
        },
        {
            "address": "bÛ>dÐ¾Ð|",
            "employees": [
                {
                    "created": "1864-04-17T00:00:01.011Z",
                    "name": null,
                    "id": 27
                },
                {
                    "created": "1864-06-07T00:00:01.011Z",
                    "name": null,
                    "id": -3
                },
                {
                    "created": "1864-04-15T00:00:01.011Z",
                    "name": null,
                    "id": -20
                },
                {
                    "created": "1864-04-14T00:00:01.011Z",
                    "name": "!Q\u0000\u0012pZ*=)I\u0013ÛbjP/K",
                    "id": 26
                },
                {
                    "created": "1864-05-02T00:00:01.011Z",
                    "name": "`^H¼IgïA:\u000cd\u001eö3DßnÇ\u0000",
                    "id": 13
                },
                {
                    "created": "1864-04-12T00:00:01.011Z",
                    "name": null,
                    "id": -10
                },
                {
                    "created": "1864-05-16T00:00:01.011Z",
                    "name": "rY,Ï¬ÿ³®=ètKH:\u0000àg\u001f",
                    "id": 24
                },
                {
                    "created": "1864-04-18T00:00:01.011Z",
                    "name": null,
                    "id": 24
                },
                {
                    "created": "1864-05-24T00:00:01.011Z",
                    "name": "W3Ý{(7!J\\\u0006o®",
                    "id": -20
                },
                {
                    "created": "1864-04-22T00:00:01.011Z",
                    "name": null,
                    "id": -25
                },
                {
                    "created": "1864-04-28T00:00:01.011Z",
                    "name": null,
                    "id": 22
                },
                {
                    "created": "1864-04-26T00:00:01.011Z",
                    "name": null,
                    "id": -2
                },
                {
                    "created": "1864-05-07T00:00:01.011Z",
                    "name": "\u0005ºbZwF\u000e\u001bBcMvt\u000cB",
                    "id": -3
                },
                {
                    "created": "1864-06-05T00:00:01.011Z",
                    "name": "m\u001ccV.<\u0011\t\u00153\u000f¿}\u0014E",
                    "id": 6
                },
                {
                    "created": "1864-06-08T00:00:01.011Z",
                    "name": "~wÅ\u001c\r\u0015\u001d",
                    "id": -25
                },
                {
                    "created": "1864-05-06T00:00:01.011Z",
                    "name": "#\u001blº_I\u0019(",
                    "id": -6
                },
                {
                    "created": "1864-05-28T00:00:01.011Z",
                    "name": null,
                    "id": 5
                },
                {
                    "created": "1864-04-09T00:00:01.011Z",
                    "name": "ËXÀR\u001f[%vMD3\u0000'",
                    "id": -23
                },
                {
                    "created": "1864-04-20T00:00:01.011Z",
                    "name": "\u0010\u000b'\u0015rYU{F1",
                    "id": 10
                }
            ]
        },
        {
            "address": "0[ª<6\u0010s!?4\u0011\u001dC O}\u000fÁ\u0004$\u0018\u0013mCgn\r_",
            "employees": [
                {
                    "created": "1864-06-08T00:00:01.011Z",
                    "name": "i\tò$C¹",
                    "id": 13
                },
                {
                    "created": "1864-05-15T00:00:01.011Z",
                    "name": "\u0003(Dk\nP\u0004Æ:+\u0011ä-\u0002ÛD\u0016ü^âdx+",
                    "id": 29
                },
                {
                    "created": "1864-04-23T00:00:01.011Z",
                    "name": "U",
                    "id": -30
                },
                {
                    "created": "1864-04-30T00:00:01.011Z",
                    "name": null,
                    "id": -27
                },
                {
                    "created": "1864-05-10T00:00:01.011Z",
                    "name": "\u001f%ã_\u0007]è»KkX.\u0007&*<Õ\u001aäA)\u0017\rb'",
                    "id": 30
                }
            ]
        },
        {
            "address": "",
            "employees": [
                {
                    "created": "1864-04-27T00:00:01.011Z",
                    "name": null,
                    "id": 21
                },
                {
                    "created": "1864-06-03T00:00:01.011Z",
                    "name": "Wm\\E2s&­h\u00190\u0003\u0019\u0011;6",
                    "id": 17
                },
                {
                    "created": "1864-04-16T00:00:01.011Z",
                    "name": "U3<\u0011.Q\u00160\u0000f\u0012[ur¡\u000bá;õz",
                    "id": 17
                },
                {
                    "created": "1864-06-05T00:00:01.011Z",
                    "name": "³qJ\u0002N7\u0002OQ\u0000v¬¦#\u0017Ñ",
                    "id": 13
                },
                {
                    "created": "1864-04-21T00:00:01.011Z",
                    "name": "\u001f/Z.\u0006èXºÆ",
                    "id": -6
                },
                {
                    "created": "1864-04-19T00:00:01.011Z",
                    "name": "-",
                    "id": 8
                },
                {
                    "created": "1864-05-16T00:00:01.011Z",
                    "name": "\u001a",
                    "id": -11
                },
                {
                    "created": "1864-06-05T00:00:01.011Z",
                    "name": "cÛ\u001fS\u0013®\u0005DEOÉ'»5<ñ°³)R1·,",
                    "id": -7
                },
                {
                    "created": "1864-04-16T00:00:01.011Z",
                    "name": "Tw\u001eÅ\t\u00147\u0012\u0019\u0001¼/[k9\u000cÅ\u0016\u0002\u0012 1Æ\u0015y'\u001c\u001f",
                    "id": -20
                },
                {
                    "created": "1864-05-04T00:00:01.011Z",
                    "name": "Û'\u0017c\u0002uGÑE:Ì",
                    "id": 17
                },
                {
                    "created": "1864-04-20T00:00:01.011Z",
                    "name": null,
                    "id": -17
                },
                {
                    "created": "1864-05-08T00:00:01.011Z",
                    "name": null,
                    "id": -3
                },
                {
                    "created": "1864-04-22T00:00:01.011Z",
                    "name": "\u0011@8ôUÓXjVrJ\"h\u0011i\"[\u0004A¼",
                    "id": 15
                },
                {
                    "created": "1864-05-16T00:00:01.011Z",
                    "name": "h0\u0013",
                    "id": 20
                },
                {
                    "created": "1864-04-19T00:00:01.011Z",
                    "name": "Î|oQ5\u001fy\u0006\u000f\u0000\u000b_¢Og",
                    "id": -19
                },
                {
                    "created": "1864-05-29T00:00:01.011Z",
                    "name": null,
                    "id": -25
                },
                {
                    "created": "1864-05-26T00:00:01.011Z",
                    "name": "Ék\u0000jd4ö",
                    "id": -1
                },
                {
                    "created": "1864-04-13T00:00:01.011Z",
                    "name": ">_$9Æ¯2\u000b¡#.e\u000fo\u0019Ç,á¯,",
                    "id": 11
                },
                {
                    "created": "1864-04-16T00:00:01.011Z",
                    "name": "kaRÑc¬\u00050Q",
                    "id": -10
                },
                {
                    "created": "1864-05-04T00:00:01.011Z",
                    "name": "\nF\u0001_ZzùÏcÈYñj2C",
                    "id": -7
                },
                {
                    "created": "1864-04-24T00:00:01.011Z",
                    "name": "",
                    "id": 4
                },
                {
                    "created": "1864-05-10T00:00:01.011Z",
                    "name": null,
                    "id": -19
                },
                {
                    "created": "1864-06-07T00:00:01.011Z",
                    "name": null,
                    "id": -23
                },
                {
                    "created": "1864-05-07T00:00:01.011Z",
                    "name": "ÃL\u000cNö.&\u0010ý\u0002£_®zyä0;µ",
                    "id": 30
                }
            ]
        },
        {
            "address": "\u000cX¹¹\u0003jt!ø",
            "employees": [
                {
                    "created": "1864-04-09T00:00:01.011Z",
                    "name": "`(ñh\u0002\rOAxê]Æ4",
                    "id": 5
                },
                {
                    "created": "1864-04-25T00:00:01.011Z",
                    "name": "ÛGJ¯7<dOb\u0006c³",
                    "id": -12
                },
                {
                    "created": "1864-05-28T00:00:01.011Z",
                    "name": "GLæë\u0011¼6!Ô¹",
                    "id": -18
                }
            ]
        },
        {
            "address": "",
            "employees": [
                {
                    "created": "1864-06-07T00:00:01.011Z",
                    "name": null,
                    "id": -4
                },
                {
                    "created": "1864-04-28T00:00:01.011Z",
                    "name": "!>\u001c,$?Ô8d",
                    "id": -3
                },
                {
                    "created": "1864-04-30T00:00:01.011Z",
                    "name": null,
                    "id": -8
                },
                {
                    "created": "1864-06-01T00:00:01.011Z",
                    "name": "q\u0003_pq7Ã¶",
                    "id": -18
                },
                {
                    "created": "1864-05-11T00:00:01.011Z",
                    "name": "Í\u001d\\v\u0007ù]\u0012\u00055ãI2",
                    "id": -1
                },
                {
                    "created": "1864-05-17T00:00:01.011Z",
                    "name": "\u0010H0V",
                    "id": 3
                },
                {
                    "created": "1864-05-09T00:00:01.011Z",
                    "name": "|\\\u0012S?ó¢\"{ST\u000e\u001dSÏ^Û5]Ã%Ï5%",
                    "id": 25
                },
                {
                    "created": "1864-05-19T00:00:01.011Z",
                    "name": "h_'5)õ*kW*pA0ô½e{\\A",
                    "id": -17
                },
                {
                    "created": "1864-05-28T00:00:01.011Z",
                    "name": null,
                    "id": -8
                },
                {
                    "created": "1864-04-30T00:00:01.011Z",
                    "name": "pV}ÿ<C\u000f52]f\u000c=?a<eW+nW~\u0008@?î\u0011",
                    "id": -9
                },
                {
                    "created": "1864-04-12T00:00:01.011Z",
                    "name": "µ\t8}W\u0011|WP^",
                    "id": -6
                },
                {
                    "created": "1864-05-24T00:00:01.011Z",
                    "name": null,
                    "id": 12
                },
                {
                    "created": "1864-05-07T00:00:01.011Z",
                    "name": "_I`\u0019:#æ{n¤@\u0003/<",
                    "id": -9
                },
                {
                    "created": "1864-06-04T00:00:01.011Z",
                    "name": null,
                    "id": -22
                },
                {
                    "created": "1864-06-06T00:00:01.011Z",
                    "name": "Q\u0013<~JC-sÏ\u0013ak\u0012Vê!\u0012@7¢eMd=\u001cn",
                    "id": 20
                },
                {
                    "created": "1864-04-28T00:00:01.011Z",
                    "name": "Ks\u0017#ÁÞ.X¸.aÁFU^C\t",
                    "id": -3
                },
                {
                    "created": "1864-04-25T00:00:01.011Z",
                    "name": "`+\u001d?",
                    "id": -26
                },
                {
                    "created": "1864-05-31T00:00:01.011Z",
                    "name": null,
                    "id": -20
                },
                {
                    "created": "1864-05-03T00:00:01.011Z",
                    "name": "Û/\u0015RcuÄj\n",
                    "id": 13
                },
                {
                    "created": "1864-04-14T00:00:01.011Z",
                    "name": null,
                    "id": 21
                },
                {
                    "created": "1864-05-11T00:00:01.011Z",
                    "name": null,
                    "id": 24
                },
                {
                    "created": "1864-04-25T00:00:01.011Z",
                    "name": null,
                    "id": 1
                }
            ]
        }
    ]
}